finagle-redis/src/main/scala/com/twitter/finagle/Redis.scala

```diff
@@ -51,6 +51,9 @@ object Redis extends Client[Command, Reply] with RedisRichClient {
       .insertBefore(DefaultPool.Role, RedisPool.module)
   }
 
+  /*
+   * ServiceFactory loaded with Command and its Reply to Stacks.
+   */
   case class Client(
       stack: Stack[ServiceFactory[Command, Reply]] = Client.newStack,
```

Comment:
`ServiceFactory` loaded with `Command` and its `Reply` to `Stack`. 

-> `finagle/finagle-core/src/main/scala/com/twitter/finagle/Stack.scala`